components/financial_assistance/app/controllers/api/v1/applications_controller.rb

```diff
@@ -0,0 +1,290 @@
+# frozen_string_literal: true
+
+module FinancialAssistance
+    class ApplicationsController < FinancialAssistance::ApplicationController
```

Comment:
This will need to be namespaced to match the routes like:
```
module FinancialAssistance
  module API
    module V1
      class ApplicationsController < FinancialAssistance::ApplicationController
```